scripts/data_catalog.py

```
@@ -0,0 +1,161 @@
+#!/usr/bin/env python3
+import sys
+from pathlib import Path
+sys.path.append(str(Path(__file__).parent.parent))
+
+from datetime import datetime
+import pandas as pd
+from typing import Dict, List, Optional
```

Ruff (0.12.2)

`typing.Dict` is deprecated, use `dict` instead

(UP035)